src/__tests__/components/EditorFooter/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import EditorFooter from "../../../components/EditorFooter";
```

Comment:
I suggest you can use "@/" instead of "../../../", it might make path more clear.